### PR TITLE
150 fix ci on deploy step

### PR DESCRIPTION
**Pull request by wazolab**
Why do we need to generate fixtures on Deploy ?

```bash
ci.yml - Line 166

- ruby poi_explode.rb cypress/fixtures/teritorio/references/pois.geojson
```

Why this uncomment here ? Can’t we leave the command uncommented ?

```bash
docker-compose.yml - Line 9

sed -i "s/# restart: unless-stopped/restart: unless-stopped/"
```

I think we could remove:
```bash
touch vidos-config.json
```
as it is generated by `yarn postinstall` right after a `yarn install`.

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -79,5 +79,5 @@ cypress
 # Vido
 **/*.stories.ts
 **/*.story.vue
-vidos.config.ts
+# vidos.config.ts
```
Comment (frodrigo) [inline .dockerignore:82]: No, we really want to ignore it.
It is local config file.

Comment (frodrigo) [reply .dockerignore:82]: Same thing, as it is generated from vidos.config.ts

Comment (wazolab) [reply .dockerignore:82]: After thinking, I don't understand why we ignore it from Docker being into the docker image build BUT we add it as a volume in docker-compose ?

Comment (wazolab) [reply .dockerignore:82]: We need `vidos.config.ts` if we want to execute `yarn build-config` which produces `vidos-config.json` file in whatever machine.

**Comment by wazolab** [comment]
My last commit adds the `yarn build-config` in the Dockerfile. As of now I am able to run docker without error on local (without docker-compose).
```bash
$ docker build -t vido . 
$ docker run -p 3000:3000 --name vido vido
```
At the end in the image we only have :
- `.output` folder
- `vidos-config.json` generated file

**Pull request closed without merging**